xml_converter/tests/test_bool.cpp

```diff
@@ -0,0 +1,70 @@
+#include "../src/attribute/bool.hpp"
+#include "../src/packaging_xml.hpp"
```

Comment:
seems strange that the "bool" module that has nothing to do with xml needs to include xml. You should refactor this out.

Reply:
Good point. While the test files are for bools, the module is really about comparing xml->xml. Renamed the tests to reflect this.